src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/adapters/FolderAdapter.js

```diff
@@ -6,20 +6,26 @@ import Pagination from '../../../components/Pagination';
 import {translate} from '../../../utils/Translator';
 import PaginatedLoadingStrategy from '../loadingStrategies/PaginatedLoadingStrategy';
 import FlatStructureStrategy from '../structureStrategies/FlatStructureStrategy';
+import type {LoadingStrategyInterface} from '../types';
+import FullLoadingStrategy from '../loadingStrategies/FullLoadingStrategy';
 import AbstractAdapter from './AbstractAdapter';
 
 @observer
 class FolderAdapter extends AbstractAdapter {
-    static LoadingStrategy = PaginatedLoadingStrategy;
-
     static StructureStrategy = FlatStructureStrategy;
 
+    static paginatable = true;
```

Comment:
not needed anymore when passinge `paginated` to the constructor of the `LoadingStrategy`